docs/building-a-form/creating-custom-fields-and-widgets.md

```diff
@@ -0,0 +1,58 @@
+# Creating custom fields and widgets
```

Comment:
This file is everything that seemed customization-related out of "Creating a form config file". Does it make sense as-is? Should any other content from other topics move into this one?

Reply:
I think this is good for now!